### ISSUE TITLE
Dependencies between issues

### ISSUE THREAD

**Issue by codingwithelijah** [amosproj/amos2022ss05-find-my-hearing-aid#27]

**Comment by cynthiachen28** [comment]
@codingwithelijah It seems not clear to me. What do you mean of issues? Issues in Product backlog or?

**Comment by codingwithelijah** [comment]
@cynthiachen28: Yes, I refer to issues in the product backlog or sprint backlog. Sometimes there are dependencies content wise. Could you check if you can link or connect certain issues? That we know: "Oh, Issue #33 belongs also to issue #22 and so on".

**Comment by cynthiachen28** [comment]
All right! I get it. Thanks.

**Comment by codingwithelijah** [comment]
@cynthiachen28: Did you already find something out? :)

**Comment by cynthiachen28** [comment]
> @cynthiachen28: Did you already find something out? :)

Yes, you can directly reference another issue in the content of a certain issue by just type "#+number of issue". 
After some research work, I found Github itself hasn't had function to manage the dependencies among issues. 

Thus, I would recommend, if two or more issues have dependencies, we could 
- add a title "Prerequisite(Dependencies)" before "User Story", and link the related issues there.
- Or add a tag shows "Prereq/Depend #1", and it would easy to filter these issues out.

**Comment by codingwithelijah** [comment]
@cynthiachen28: Great! Thank you very much, this helps a lot :)
